manifests/databases.pp

```diff
@@ -1,11 +1,17 @@
-define pgbouncer::databases( 
+# == define: pgbouncer::databases
+#
+# defined type that can be used to create the databases section in the config
+# file
+#
+define pgbouncer::databases(
   $databases = [],
 ) {
 
   validate_array($databases)
 
   unless empty($databases[0]) {
-    concat::fragment { "${databases[0]['dest_db']}_${databases[0]['auth_user']}":
+    concat::fragment {
+    "${databases[0]['dest_db']}_${databases[0]['auth_user']}":
```

Comment:
When trying to make this line less than 80 characters, the preferred way is to set the quoted value into a variable and then use the variable in the `concat::fragment` title.  Putting a new line is not preferred.

Example:
```puppet
$my_unique_title = "${databases[0]['dest_db']}_${databases[0]['auth_user']}"
concat::fragment { $my_unique_title:
...
```